### PR TITLE
docs(top): change in usages

### PR DESCRIPTION
**Pull request by ahmadawais**
Thank you for such a fantastic framework which allowed us to build both a local and Serverless AI server.

Adds the open-source web AI framework [BaseAI.dev](https://BaseAI.dev) as we're using Hono and Cloudflare heavily.

Asked @yusukebe before opening this PR.

image

P.S. Also honored to be part [The story of web framework Hono, from the creator of Hono](https://blog.cloudflare.com/the-story-of-web-framework-hono-from-the-creator-of-hono/)

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -74,6 +74,7 @@ Here are some examples of use-cases.
 | ------------------------------------------------------------------ | ------------------ | --------------------------------------------------------------------------------------- |
 | [cdnjs](https://cdnjs.com)                                         | Cloudflare Workers | A free and open-source CDN service. _Hono is used for the api server_.                  |
 | [Cloudflare D1](https://www.cloudflare.com/developer-platform/d1/) | Cloudflare Workers | Serverless SQL databases. _Hono is used for the internal api server_.                   |
+| [BaseAI](https://baseai.dev)                                       | Local AI Server    | Serverless AI agent pipes with memory. An open-source agentic AI framework for web. _API server with Hono_. |
```
Comment (yusukebe) [inline docs/index.md:77]: How about writing just `Local server` instead of `Local AI Server`?

**Comment by yusukebe** [comment]
Hi @ahmadawais 

Thank you for the PR. I've left a comment!

File: docs/index.md (hunk 1 above)
Comment (ahmadawais) [reply docs/index.md:77]: Works for me. My thinking was like this: creating a `Local server` is pretty easy. Creating a `Local AI Server` is quite hard. That's where Hono helped us. 

If you still want me to make it `Local server`, then I can send the fix right away. 

Let me know what you think.

Comment (yusukebe) [reply docs/index.md:77]: @ahmadawais 

Understood well! Let's go with this.

**Comment by yusukebe** [comment]
@ahmadawais 

Thanks. Merging now.

File: docs/index.md (hunk 1 above)
Comment (ahmadawais) [reply docs/index.md:77]: Thank you!! 💜

**Pull request merged**